tests/users/test_api_resend_email.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+from flask import json
+
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from tests.test_data import user1
+
+
+class TestResendEmailApi(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.verified_user = UserModel(**user1)
+
+        self.verified_user.is_email_verified = True
+        self.verified_user.is_available = False
+        db.session.add(self.verified_user)
+        db.session.commit()
+
+    # This function checks if the status code returns 409 for exisiting email
+    def test_error_code_for_resend_email(self):
```

Comment:
Use descriptive names
```suggestion
    def test_resend_email_user_already_verified(self):
```